### PR TITLE
Add map with parks, zoom and popups on click 

### PR DESCRIPTION
**Pull request by lyzidiamond**

### CODE REVIEW

File: app/assets/javascripts/parks.js.jsx [hunk 1]
```diff
@@ -62,7 +62,7 @@ var ParksList = React.createClass({
           </ol>
         </div>
         <div className="large-8 columns">
-          <img src="http://4.bp.blogspot.com/-O6IR1WmyEd8/UZ-APOBLFFI/AAAAAAAAEfE/lONvEFkWW-Y/s1600/Lexington+KY+Birds+eye.jpg" width="100%" />
+          <div id='map'></div>
```
Comment (eeeschwartz) [inline app/assets/javascripts/parks.js.jsx]: Let's use double quotes in the name of uniformity.

File: app/assets/stylesheets/_custom-styles.scss [hunk 2]
```diff
@@ -1,3 +1,7 @@
 .border-bottom {
   border-bottom: 1px solid rgba(0,0,0,0.10);
 }
+
+#map {
+  height: 400px;
+}
```
Comment (eeeschwartz) [inline app/assets/stylesheets/_custom-styles.scss]: Would you mind to add newline at end of file?

Sublime can do automatically with the `ensure_newline_at_eof_on_save` [setting](http://sublime-text-unofficial-documentation.readthedocs.org/en/latest/reference/settings.html#automatic-behavior) in preferences

File: app/views/parks/index.html.erb [hunk 3]
```diff
@@ -1 +1 @@
-<div id="content"></div>
+<div id="content"></div>
```
Comment (eeeschwartz) [inline app/views/parks/index.html.erb]: Newline warning here as well

File: app/assets/javascripts/application.js [hunk 4]
```diff
@@ -15,6 +15,8 @@
 //= require foundation
 //= require react
 //= require lodash
-//= require_tree .
```
Comment (eeeschwartz) [inline app/assets/javascripts/application.js:18]: I think we can actually revert this piece and use require_tree. I reckon we solved the code-order problem by running the map js after the dom load and React rendering.

**Comment by lyzidiamond** [comment]
All changed.

**Pull request merged**